website/app/page.tsx

```diff
@@ -1,12 +1,44 @@
 'use client';
 import { Button } from '@/components/ui/button';
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectLabel,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select"
 import { Toaster, toast } from 'alert';
+import { useState } from 'react';
+
+type ToasterPosition = 'top-left' | 'top-center' | 'top-right' | 'bottom-left' | 'bottom-center' | 'bottom-right';
 
 export default function App() {
+  const [position, setPosition] = useState<ToasterPosition>('bottom-right');
+  
   return (
     <div className="w-screen h-screen flex justify-center items-center">
-      <Toaster />
+      <Toaster position={position}/>
       <div className="flex gap-4">
+        <Select onValueChange={(s) => {
```

Comment:
```suggestion
        <Select value={position} onValueChange={(s) => {
```

Reply:
Hey @AdityaNik!
I appreciate your contribution – it's exactly what I was looking for. Could you please make a small adjustment so that the selection is initially set? Once that's done, this PR will be good to merge!

Reply:
Hey @AdityaNik!

I appreciate your contribution – it's exactly what I was looking for. Could you please make a small adjustment so that the select is initially set? Once that's done, this PR will be ready to merge!